src/InteractiveWindow/Editor/InteractiveWindow.UIThreadOnly.cs

```diff
@@ -474,12 +474,36 @@ public void InsertCode(string text)
                 }
                 else
                 {
-                    if (!TextView.Selection.IsEmpty)
+                    using (var transaction = UndoHistory_TestOnly.CreateTransaction(InteractiveWindowResources.TypeChar))
```

Comment:
Hmm? Is "UndoHistory_TestOnly" only used in tests? If so, how? Is there no undo in the product here?